rootfs/ingest/python/qserv/jsonparser.py

```diff
@@ -77,18 +77,20 @@ class TransactionState(Enum):
 
 
 class ContributionMonitor:
-    """Store contribution status returned by the Ingest Service
-    see https://confluence.lsstcorp.org/display/DM/Ingest%3A+9.5.3.+Asynchronous+Protocol
-    for details
+    """Store contribution status returned by the Ingest Service see https://con
+    fluence.lsstcorp.org/display/DM/Ingest%3A+9.5.3.+Asynchronous+Protocol for
+    details.
 
     Parameters
     ----------
-        response_json: `dict`
-            Ingest Service response for a HTTP request against URL: ingest/file-async/<request_id>
+    response_json: `dict`
+        Ingest Service response for a HTTP request
+        against URL: ingest/file-async/<request_id>
 
     Raises
     ------
-        ReplicationControllerError: If 'response_json' does not have the expected value or format
+    error: `ReplicationControllerError`
```

Comment:
Just `ReplicationControllerError`.

Please review the style guide https://developer.lsst.io/python/numpydoc.html and try to follow that.